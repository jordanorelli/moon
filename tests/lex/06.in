[]
[1 2 3]
["string one" "string two"]
[[1 2 3][]["string one" "string two"]]
