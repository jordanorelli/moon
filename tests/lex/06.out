{t_list_start [}
{t_list_end ]}
{t_list_start [}
{t_real_number 1}
{t_real_number 2}
{t_real_number 3}
{t_list_end ]}
{t_list_start [}
{t_string string one}
{t_string string two}
{t_list_end ]}
{t_list_start [}
{t_list_start [}
{t_real_number 1}
{t_real_number 2}
{t_real_number 3}
{t_list_end ]}
{t_list_start [}
{t_list_end ]}
{t_list_start [}
{t_string string one}
{t_string string two}
{t_list_end ]}
{t_list_end ]}
